MobileCenter/MobileCenterTests/MSChannelDefaultTests.m

```diff
@@ -79,7 +79,9 @@ - (void)testEnqueuingItemsWillIncreaseCounter {
 
   // When
   for (int i = 1; i <= itemsToAdd; i++) {
-    [self.sut enqueueItem:[MSAbstractLog new] withCompletion:nil];
+    id logMock = OCMPartialMock([MSAbstractLog new]);
+    OCMStub([logMock isValid]).andReturn(YES);
```

Comment:
Yes.. Good point. I thought it is good to repeat because there is no context that `isValid` is mocked if we use a method in test. Now changed my mind and agree with you.